sdf.md

```diff
@@ -1077,8 +1080,20 @@ The keyword "sdfRequired" is provided to apply a constraint that
 defines for which declarations the corresponding data are mandatory in a
 Thing modeled by the current definition.
 
-The value of "sdfRequired" is an array of name references (JSON pointers), each
-indicating one declaration that is mandatory to be represented.
+The value of "sdfRequired" is an array of references, each indicating
+one or more declarations that are mandatory to be represented.
+
+References in this array can be SDF names (JSON Pointers), or one of
+two abbreviated references:
+
+* a text string with an affordance name or grouping name.
+All affordance declarations that are in the same grouping and that
```

Comment:
Suggest to limit this to affordances and groupings that are directly part of the grouping where the sdfRequired is 
```suggestion
All affordance declarations that are directly (i.e., not nested further in another grouping) in the same grouping and that
```